core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -378,4 +391,48 @@ public void testSpatialId() {
         assertEquals(5, encoder.getMaxspeed(e3, -1, false), .1);
         assertEquals(-1, encoder.getMaxspeed(e4, -1, false), .1);
     }
+
+    /**
+     * Defines the default rules for German roads
+     *
+     * @author Robin Boldt
+     */
+    private static class TestRule extends DefaultSpatialRule {
```

Comment:
It should only contain what's necessary for the test (feel free to reduce). I didn't want to use Germany because I wanted to move Germany out of the core and make it private, to illustrate that the two countries together with their factory are self-contained.